### DISCUSSION TITLE
OpenWRT mobile routers, like GLInet

### DISCUSSION THREAD

**Discussion by HeeroML** [SmoothWAN/SmoothWAN#19] [General & Help]
Will this work in these setups? Want to use a glinet  beryl for that, as I already have it.

**Comment by TalalMash** [comment]
> Will this work in these setups? Want to use a glinet beryl for that, as I already have it.

Brume-W may work well up to ~100Mbit from my estimation. (AES acceleration is supported)
The Beryl CPU is MIPS and it's very slow, and MIPS is unsupported.

Edit: Checkout the Slate AX support page.

**Comment by techtaxi** [comment]
I would love to something like this for my Opal GL-Inet router as well. Even if it's not real fast, I just need to maintain 40mbps down and up at most for twitch streaming and watching streams.

**Reply by TalalMash** [reply]
Unfortunately, Opal has SF19A28 SoC which includes MIPS CPU architecture, the current Speedify release targets CPUs with `arm`, `arm64` and `amd64`.

**Reply by techtaxi** [reply]
any way to get a release for mips ?

**Reply by TalalMash** [reply]
You can ask them, but bonding is quite demanding, adding more WANs can reduce the speed and increase the latency to unusable levels with a very low end spec.
https://support.speedify.com/article/525-router-switch

**Comment by TalalMash** [comment]
GL.iNet Slate AX support added.
